docs/en/configuration.rst

```diff
@@ -186,6 +186,23 @@ file:
 
     export PHINX_ENVIRONMENT=dev-`whoami`-`hostname`
 
+Migration Table
+---------------
+
+To keep track of the migration statuses for an environment, phinx creates
+a table to store this information. You can customize where this table
+is created by configuring ``default_migration_table``:
+
+.. code-block:: yaml
+
+    environment:
+        default_migration_table: phinxlog
+
+If this field is omitted, then it will default to ``phinxlog``. For
+databases that support it, e.g. Postgres, the schema name can be prefixed
+with a period separator (``.``). For example, ``phinx.log`` will create
+the table log in the ``phinx`` schema instead of ``phinxlog`` in the
```

Comment:
`log` should have been ` ``log`` `.

Reply:
Thank you, resolved in https://github.com/cakephp/phinx/commit/96c731595faff67dbcca2c303c067fcf219d226a